content/CORE/CORETutorials/Services/ConfiguringLLDP.md

```diff
@@ -1,7 +1,10 @@
 ---
 title: "Configuring LLDP"
+description: "Configuring Link Layer Discovery Protocol (LLDP)"
```

Comment:
Description comment is still too close to the title. Try a complete sentence like, "This article provides instructions on configuring Link Layer Discovery Protocol (LLDP) on your TrueNAS." You can use describes how to configure or set up, or some variation on the theme.